### PR TITLE
Tidying up a lot of code. Moving api url definitions in to oj-url-helper...

### PR DESCRIPTION
**Pull request by stuartlynn**
...s. 

Have also added some helper elements for JSON post and get

### CODE REVIEW

**Comment by stuartlynn** [comment]
@marcrohloff  Sorry for the self merge. I think this is ok though and I need some of this stuff to move on. If anything isn't ok I am happy to change it.

File: app/assets/components/elements/json-post.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  <script>
+    Polymer({
+      auto: false,
+      data: {},
+      go:function(){
+        console.log("data is ", this.data)
+        this.$.request.go()
+      },
+      success:function(event,detail){
+        this.fire("success", detail)
```
Comment (marcrohloff) [inline app/assets/components/elements/json-post.html]: Is there a reason the parameters here are different to the ones in the get version?

**Comment by stuartlynn** [comment]
ack no I will fix that up

File: app/assets/components/subpages/oj-review-page.html [hunk 2]
```diff
@@ -10,9 +10,12 @@
       </template>
       <span><core-icon icon='list'></core-icon></span>
 
-      <paper-progress  class='bottom fit' value="{{loadProgress}}"></paper-progress>
+      <paper-progress class='bottom fit' value="{{loadProgress}}"></paper-progress>
     </core-toolbar>
 
+    <oj-editor-paper-controlls flex paper={{paper}}></oj-editor-paper-controlls>
```
Comment (marcrohloff) [inline app/assets/components/subpages/oj-review-page.html]: I don't see the definition for this. Also ```controlls``` is spelled incorrectly.

**Pull request merged**